src/prefect/server/api/automations.py

```diff
@@ -244,14 +244,16 @@ async def read_automations(
     sort: AutomationSort = Body(AutomationSort.NAME_ASC),
     limit: int = LimitBody(),
     offset: int = Body(0, ge=0),
+    automations: Optional[AutomationFilter] = None,
```

Comment:
Needed to rename the automations module since otherwise adding it as a param here overrides the endpoint logic below